src/organisms/cards/StackedCard/index.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classnames from 'classnames';
+import styles from './stacked_card.module.scss';
+
+const stackChild = child => <li className={styles['stacked-li']}>{ child }</li>;
+const stackChildren = children => React.Children.map(children, stackChild);
+
+function StackedCard(props) {
+  const {
+    children,
+    className
+  } = props;
+
+  return (
+    <ul className={classnames(styles['stacked'], className)}>
```

Comment:
one other thing you can test for is that props.className gets appended to this div - this might be overkill so lmk what you think!